webpack build should hard fail in production

htuscher opened the pull request:
If you build your administration with nodejs you should be able to work 
with exit codes. Without bail it's always 0

<!--
Thank you for contributing to Shopware! Please fill out this description template to help us to process your pull request.

Please make sure to fulfil our contribution guideline (https://docs.shopware.com/en/shopware-platform-dev-en/contribution/contribution-guideline?category=shopware-platform-dev-en/contribution).

Do your changes need to be mentioned in the documentation?
Add notes on your change right now in the documentation files in /src/Docs/Resources and add them to the pull request as well. 
-->

### 1. Why is this change necessary?


### 2. What does this change do, exactly?


### 3. Describe each step to reproduce the issue or behaviour.


### 4. Please link to the relevant issues (if any).


### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

OliverSkroblin commented:
Hi @shyim,
thanks for the pull request.
The changes are merged and will be available with version 5.3.5.

Commit:
webpack build should hard fail in production

If you build your administration with nodejs you should be able to work 
with exit codes. Without bail it's always 0

raknison commented:
Hey @hhoechtl , thanks for your contribution 💙
